src/browser/NativeMessageInstaller.cpp

```diff
@@ -58,6 +59,8 @@ namespace
         QStringLiteral("HKEY_CURRENT_USER\\Software\\Chromium\\NativeMessagingHosts\\org.keepassxc.keepassxc_browser");
     const QString TARGET_DIR_FIREFOX =
         QStringLiteral("HKEY_CURRENT_USER\\Software\\Mozilla\\NativeMessagingHosts\\org.keepassxc.keepassxc_browser");
+    const QString TARGET_DIR_LIBREWOLF =
+        QStringLiteral("HKEY_CURRENT_USER\\Software\\Mozilla\\NativeMessagingHosts\\org.keepassxc.keepassxc_browser");
```

Comment:
LibreWolf really uses Mozilla's path on Windows?

Reply:
Yes, I tested it. Works out of the box and also works while FF and LW are both using the addons on the same DB at the same time.

Reply:
I didn't say to change it back, you had originally made the PR with Mozilla. I told you to assign the LIBREWOLF variable to the Mozilla one to make it explicit that they were the same path.

Reply:
Then I misunderstood your comment; at the time you commented, I already made the path unique. At first I used the same path, because that also works, but the LW team decided a unique path would be better.

TLDR; can we just use the unique path?

Reply:
Thank you for the approval, I'm sure it will work. The json file will still be the same, at least that's how I understand it, so there shouldn't be any problem.